lib/gridstore/src/tracker.rs

```diff
@@ -222,18 +223,16 @@ impl Tracker {
         let start_offset =
             size_of::<TrackerHeader>() + point_offset * size_of::<Option<ValuePointer>>();
         let end_offset = start_offset + size_of::<Option<ValuePointer>>();
-        // check if file is long enough
+
+        // Grow tracker file if it isn't big enough
         if self.mmap.len() < end_offset {
-            // flush the current mmap
             self.mmap.flush().unwrap();
-            let missing_space = end_offset - self.mmap.len();
-            // reopen the file with a larger size
-            // account for missing size + extra to avoid resizing too often
-            let new_size = self.mmap.len() + missing_space + Self::DEFAULT_SIZE;
+            let new_size = end_offset.next_power_of_two();
```

Comment:
This now grows the tracker file by a power of two. It means the tracker file will have a size of: 1MB, 2MB, 4MB, 8MB...

Before the number was quite arbitrary, not being round numbers.

While it is not strictly required, I like the round numbers better.

Reply:
I like powers of two

Reply:
:zap:.:zap: